assets/css/app.scss

```diff
@@ -27,6 +27,7 @@
 @import "components/app-header";
 @import "components/at-mention";
 @import "components/box";
+@import "components/buttons";
```

Comment:
Since we are not styling buttons in this partial, how do you feel about naming the partial `flex-button-container`?